pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1339,7 +1339,7 @@ func TestSelectNodesForPreemption(t *testing.T) {
 			name: "a pod that fits on both machines when lower priority pods are preempted",
 			registerPlugins: []st.RegisterPluginFunc{
 				st.RegisterQueueSortPlugin(queuesort.Name, queuesort.New),
-				st.RegisterFilterPlugin(noderesources.FitName, noderesources.NewFit),
+				st.RegisterPluginAsExtensions(noderesources.FitName, 1, noderesources.NewFit, "Filter", "PreFilter"),
```

Comment:
The weight here is confusing, in a followup PR I suggest we change ```RegisterPluginAsExtensions``` to not set the weight, and have another function ```RegisterPluginAsExtensionsWithWeight``` that accept a weight parameter for the extensions that require a weight.

Reply:
I'll send another PR for this.